explore/src/main/scala/explore/config/SequenceTable.scala

```diff
@@ -116,60 +118,63 @@ object SequenceTable {
         )
 
       def rightAligned(value: Any) =
-        <.div(^.textAlign.right)(value.toString).rawElement
+        <.div(^.textAlign.right)(value.toString)
 
       val columns = List(
         StepTable
-          .Column(
-            "atomSteps",
-            _.firstOf.map((drawBracket _).andThen(_.rawElement)).orNull
-          )
-          .setHeader(" "),
+          .Column("atomSteps", _.firstOf)
+          .setHeader(" ")
+          .setCell(_.value.map(drawBracket)),
         StepTable
-          .Column("stepType", _.step.stepType.toString)
-          .setHeader("Type"),
+          .Column("stepType", _.step.stepType)
+          .setHeader("Type")
+          .setCell(_.value.toString),
         StepTable
-          .Column("exposure", s => rightAligned(s.exposureSecs))
-          .setHeader(rightAligned("Exp (sec)")),
+          .Column("exposure", _.exposureSecs)
+          .setHeader(rightAligned("Exp (sec)").rawElement)
+          .setCell(cell => rightAligned(cell.value)),
         StepTable
-          .Column(
-            "guide",
-            s => if (s.guided) Icons.Crosshairs.copy(clazz = ExploreStyles.StepGuided).raw else null
-          )
-          .setHeader(""),
+          .Column("guide", _.guided)
+          .setHeader("")
+          .setCell(cell =>
+            if (cell.value) Icons.Crosshairs.copy(clazz = ExploreStyles.StepGuided) else EmptyVdom
```

Comment:
Will `Icons.Cros...).when(cell.value)` work?

Reply:
Unfortunately, it doesn't, since `when` returns a `TagMod` and we need a `VdomNode` here.